src/pages/settings/Wallet/WalletPage/WalletPage.js

```diff
@@ -311,6 +312,8 @@ function WalletPage({bankAccountList, betas, cardList, fundList, isLoadingPaymen
 
     // Determines whether or not the modal popup is mounted from the bottom of the screen instead of the side mount on Web or Desktop screens
     const isPopoverBottomMount = anchorPosition.anchorPositionTop === 0 || isSmallScreenWidth;
+    const alertTextStyle = [styles.inlineSystemMessage, styles.flexShrink1];
+    const alertViewStyle = [styles.flexRow, styles.alignItemsCenter, styles.w100, styles.ph5];
```

Comment:
Coming from https://github.com/Expensify/App/issues/44773
Additional horizontal padding (`styles.ph5`) for `alertViewStyle` wasn't needed, it resulted in incorrect padding for Onfido messages